tests/api/utils/test_scheduler.py

```diff
@@ -111,7 +114,11 @@ async def test_create_schedule(db: Session, scheduler: Scheduler):
         bump_counter,
         cron_trigger,
     )
-    await asyncio.sleep(1 + expected_call_counter)
+
+    # cronjob starts on start of each second and until it gets to run the job it takes a few microseconds,
+    # to avoid the transient errors we round the seconds to wait and add an epsilon to each sleep.
```

Comment:
```suggestion
    # The trigger is defined with `second="*/1"` meaning it runs on round seconds, but executing the actual functional code - bumping the counter - happens a few microseconds afterwards. 
    # To avoid transient errors on slow systems, we sleep until the round second plus some extra margin
```